src/Configuration/Config/src/ConfigurationKeyComparer.cs

```diff
@@ -41,7 +41,7 @@ public int Compare(string x, string y)
                 var xIsInt = x != null && int.TryParse(x, out value1);
```

Comment:
```c#
var xIsInt = int.TryParse(x, out int value1);
var yIsInt = int.TryParse(y, out int value2);
```

The null-check is done in `TryParse`, so here it is redundant. Variable initialized by the `out` directly.